### DISCUSSION TITLE
[FR] Entry - Always keep the "first draft" after "Create", and make "Publish" a draft the only way to update "Current"

### DISCUSSION THREAD

**Discussion by hongc-cc** [craftcms/cms#8905] [Ideas]
### Description
Edited for clarity.

To make AX more consistent, when clicking "create" (i.e publish), don't convert the "first draft" to "current".
Instead, keep the "first draft", and remove the functions of directly editing the "current" version.

In terms of UX, to make changes on "Current" , the actions required are the same:
**On editing "current" and "save"**
Select "Current" > Edit fields > "Save"
**On editing "draft" and "publish"**
Select "Draft" > Edit fields > "Publish"

However, the benefit on forcing "draft" and "publishing" is:
1. Editors are more confident editing as they are always working on a draft and always have autosaving, and any saving action wont affect live version and therefore avoid accidental live changes
2. consistent UX when creating a new entry (First draft autosaving) and editing existing entry (Existing draft autosaving)
3. no more draft shfting mentioned on #6681 
4. UX can now always show the diff between "draft" & "current"
(the UI that shown when a "merge" is recommanded.)

**Comment by brandonkelly** [comment]
Related: #5661
